docs-partials/builder/azure/arm/Config-not-required.mdx

```diff
@@ -232,6 +232,11 @@
   See [documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/user-data)
   to learn more about user data.
 
+- `custom_script` (string) - Specify a command to inject into the CustomScriptExtension, to run on startup
```

Comment:
should it be customscript_command?

Reply:
Also a sample will be great somewhere, as the quotes are hard to get right :)

Reply:
Are you suggesting a rename? 
Or are you thinking this is an mistake?

If it's the second one, no it's meant to be `custom_script`.

Reply:
I've added an example. Let me know your thoughts on it.